airflow/dags/dags.py

```diff
@@ -150,9 +60,9 @@ def log_failure_to_slack(context):
         latest_only=False,
         user_defined_macros={
             **user_defined_macros,
-            "scd_join": scd_join,
-            "sql_enrich_duplicates": sql_enrich_duplicates,
-            "sql_airtable_mapping": __macros__.generate_airtable_mapping_table.generate_sql,
+            "scd_join": __macros__.sql_gtfs_schedule_scd_join.scd_join,
+            "sql_enrich_duplicates": __macros__.sql_payments_enrich_duplicates.sql_enrich_duplicates,
+            "sql_airtable_mapping": __macros__.sql_airtable_generate_mapping_table.generate_sql,
```

Comment:
Maybe these can also be added via the `**` command:

```suggestion
            **data_infra_macros
```

Reply:
Refactored again, now you don't have to make any changes to `dags.py` to add a macro (which seems good?)